pySDC/implementations/problem_classes/LeakySuperconductor.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, problem_params, dtype_u=mesh, dtype_f=mesh):
             self.dx = 1.0 / (self.params.nvars + 1)
             xvalues = np.array([(i + 1) * self.dx for i in range(self.params.nvars)])
         elif self.params.bc == 'neumann-zero':
-            self.dx = 1.0 / (self.params.nvars - 1)
+            self.dx = 1.0 / (self.params.nvars + 1)
             xvalues = np.array([(i + 1) * self.dx for i in range(self.params.nvars)])
```

Comment:
You're sure these `xvalues` are correct? It looks like they run from `1/(N+1)` to `(N+1)/(N+1)` and I guess you're missing the `0` here.

Reply:
Better now? We want to include both boundaries, a.k.a. 0 and 1, right?